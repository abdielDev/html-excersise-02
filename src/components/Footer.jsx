import React from 'react';
import '../styles/components/Footer.css';

const Footer = () => {
  return (
    <div className="Footer">
      <div className="SocialNetwork">
        <ul>
          <li>
            <i className="fa fa-twitter"></i>
          </li>
          <li>
            <i className="fa fa-facebook"></i>
          </li>
          <li>
            <i className="fa fa-whatsapp"></i>
          </li>
          <li>
            <i className="fa fa-youtube"></i>
          </li>
        </ul>
      </div>
      <small>&copy; 2012 Todos los derechos reservados</small>
    </div>
  );
};

export default Footer;