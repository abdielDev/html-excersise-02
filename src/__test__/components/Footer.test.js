import React from 'react';
import { shallow, mount } from 'enzyme';
import { create } from 'react-test-renderer';
import Footer from '../../components/Footer';

describe('<Footer />', () => {
  const footer = shallow(<Footer />);
  test('Render del componente Footer', () => {
    expect(footer.length).toEqual(1);
  });

  test('Render de los iconos de redes sociales', () => {
    expect(footer.contains(<i className="fa fa-twitter"></i>)).toBeTruthy();
  });
});

describe('Footer Snapshot', () => {
  test('Comprobar la UI del componente Footer', () => {
    const footer = create(<Footer />);
    expect(footer.toJSON()).toMatchSnapshot();
  });
});