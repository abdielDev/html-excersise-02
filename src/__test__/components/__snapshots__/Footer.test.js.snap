// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshot Comprobar la UI del componente Footer 1`] = `
<div
  className="Footer"
>
  <div
    className="SocialNetwork"
  >
    <ul>
      <li>
        <i
          className="fa fa-twitter"
        />
      </li>
      <li>
        <i
          className="fa fa-facebook"
        />
      </li>
      <li>
        <i
          className="fa fa-whatsapp"
        />
      </li>
      <li>
        <i
          className="fa fa-youtube"
        />
      </li>
    </ul>
  </div>
  <small>
    Â© 2012 Todos los derechos reservados
  </small>
</div>
`;
